web/templates/web/demographic-data-update.html

```diff
@@ -12,25 +12,89 @@
 {% block content %}
     <script>
         $(document).ready(function() {
-            if ($('#id_country')[0].value !== 'US') {
-                hideField('state');
-            }
+        
+			const gdpr_country_list = [	
```

Comment:
Would there be any benefit to pulling this list from a third-party source that updates if this country list changes?  I see you're also adding "", NO, IS, LI, so maybe not.

Reply:
Probably, I just didn't find a good source. I suspect as GDPR takes effect this will be a fairly common need and there will be a good option, so I'll keep that in mind.